sonar-vbnet-plugin/src/main/resources/documentation.md

```diff
@@ -38,7 +38,7 @@ dotnet_diagnostic.Sxxx.severity = none
 Currently an error will be thrown when an issue is raised on a line of code containing the following pattern `\s+error\s*:` (i.e. one or more spaces, the string 'error', zero or more spaces and a ':' ) . This is a well known problem on the Microsoft side (see [issue](https://github.com/dotnet/roslyn/issues/5724/)). In order to work around this problem, analysis will skip issues reported on any line where the pattern is detected.
 
 ## Related Pages
-* [Investigating the performance of .NET Analysis](https://community.sonarsource.com/t/the-sonarsource-guide-for-investigating-the-performance-of-net-analysis/47279)
+* [Investigating the performance of .NET Analysis](https://community.sonarsource.com/t/47279)
```

Comment:
shouldn't this be near `Troubleshooting guide for .NET code coverage import` ?

Reply:
@peeter-piegaze-sonarsource ping?

Reply:
Sorry for the late reply....what do you mean?

Reply:
you've added it above `* [Importing External Issues]`, however semantically it makes more sense to be below `* [Troubleshooting guide for .NET code coverage import]`

to keep similar information together

Reply:
Oh I understand. Sure. I will move it